Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,45 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "without-test", "Skip running build-time tests (not recommended)"
+  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+
+  depends_on "robodoc" => :recommended
```

Comment:
ah, OK, I had that before but for some reason the output of `brew info json-fortran` led me to think that I was doing something non-standard, reverting to this now